src/main/java/net/openhft/chronicle/queue/reader/ChronicleHistoryReader.java

```diff
@@ -160,7 +160,7 @@ public Map<String, Histogram> readChronicle() {
         final WireParselet parselet = parselet();
         final FieldNumberParselet fieldNumberParselet = (methodId, wire) -> parselet.accept(methodIdToName(methodId), wire.read());
         final MessageHistory prev = MessageHistory.get();
-        MessageHistory.set(new VanillaMessageHistory());
+        MessageHistory.emptyHistory();
```

Comment:
Hi Peter - does this need to change back?

Reply:
It could be changed back but this is much cleaner

Reply:
You're right - I've seen the implementation of `emptyHistory` now - makes sense